libr/anal/fcn.c

```diff
@@ -1516,9 +1516,13 @@ R_API int r_anal_fcn_cc(RAnal *anal, RAnalFunction *fcn) {
 
 R_API char *r_anal_function_get_signature(RAnalFunction *function) {
 	RAnal *a = function->anal;
+	char *fcnname = (r_str_startswith (function->name, "sym.imp."))
```

Comment:
actually you can do const char * and avoid the strdup, just use fcn->name + prefixskip and drop the free